src/main/java/com/example/bookstore/mapper/BookMapper.java

```diff
@@ -1,14 +1,24 @@
 package com.example.bookstore.mapper;
 
-import com.example.bookstore.config.MapperConfig;
 import com.example.bookstore.dto.BookDto;
 import com.example.bookstore.dto.CreateBookRequestDto;
 import com.example.bookstore.model.Book;
+import org.mapstruct.BeanMapping;
+import org.mapstruct.InjectionStrategy;
 import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValueCheckStrategy;
+import org.mapstruct.NullValuePropertyMappingStrategy;
 
-@Mapper(config = MapperConfig.class)
+@Mapper(componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl")
 public interface BookMapper {
     BookDto toDto(Book book);
 
     Book toBook(CreateBookRequestDto requestDto);
+
+    @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
+    void upgradeBook(CreateBookRequestDto requestDto, @MappingTarget Book book);
```

Comment:
terminology matters :)
```suggestion
    void updateBook(CreateBookRequestDto requestDto, @MappingTarget Book book);
```